agent/engine/docker_container_engine.go

```diff
@@ -623,6 +623,13 @@ func (dg *dockerGoClient) ContainerEvents(ctx context.Context) (<-chan DockerCon
 				// No interest in image events
 				continue
 			default:
+				if len(event.Status) >= 12 && event.Status[0:12] == "exec_create:" {
+					continue
+				}
+				if len(event.Status) >= 11 && event.Status[0:11] == "exec_start:" {
+					continue
+				}
+
```

Comment:
You can use the strings.HasPrefix[1] to search for prefixes. See below:
...
if strings.HasPrefix(event.Status, "exec_create:") {
continue
}
else if strings.HasPrefix(event.Status, "exec_start:"){
continue
} 
...

[1] https://golang.org/pkg/strings/#HasPrefix